pkgs/by-name/co/construct/package.nix

```diff
@@ -0,0 +1,37 @@
+{ llvmPackages
```

Comment:
`llvmPackages` is not referenced anywhere in this package, what am I missing that this is necessary?